src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,33 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * Add your docs here.
+ */
+public class Drivetrain extends Subsystem {
+  public static void Driver_Control(TalonSRX Starboard, TalonSRX Port, Joystick Driverstick)
+  {
+    Double thro = Driverstick.getRawAxis(1); //populates "thro" as the angle of the driverstick
```

Comment:
Try adding in the yaw, you can use the code in joe-drivetrain as an example, the yaw axis on the controller is number `2`